source/plugin/Assets/GoogleMobileAds/Common/RewardedAdDummyClient.cs

```diff
@@ -52,6 +52,10 @@ public void CreateRewardedAd(string adUnitId)
         public void LoadAd(AdRequest request)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
+
+            var onAdLoaded = OnAdLoaded;
+            if (onAdLoaded != null)
```

Comment:
please add { } braces around this if statement as per the conventions in this codebase.